### ISSUE TITLE
why does use_tic needs circle if I'm not using circle activation?

### ISSUE THREAD

**Issue by maelle** [ropensci/tic#206]
This might be a silly question :-)

**Comment by maelle** [comment]
oh it _was_ silly, the default seems to activate everything.

**Comment by pat-s** [comment]
In what kind if way do you mean "need" here? In the function, it should only use `circle::fun` if you specified that you want to deploy on circle:

https://github.com/ropenscilabs/tic/blob/fbcdd89f17786b320c64ebabadf99e8f7e19bff0/R/use_tic.R#L135-L139

**Comment by maelle** [comment]
yes, that's why my question was silly, I thought the default when running tic::use_tic() was not to use circle so when I got the error about circle's not being installed I got confused.

**Comment by pat-s** [comment]
By default the "wizard" is used and {circle} should only be called when you selected that you want to deploy on circle ci. 

Your questions sounds that you did not do that (i.e. only used Travis) and still got a call to `circle::fun()`?

**Comment by maelle** [comment]
I simply ran `tic::use_tic()` and got an error about circle not being installed, if I remember correctly.

**Comment by pat-s** [comment]
Ok, without a reprex I cannot reproduce this. Maybe you really selected "circle" at some decision point if you used the interactive wizard for setting up everything.

Let me know if it happens again - with the options you selected :)

**Comment by maelle** [comment]
I uninstalled circle and got

```r
> tic::use_tic(wizard=FALSE)
→ Welcome to tic!
This wizard will guide you through the setup process for getting started with various CI
providers.

── Introduction: ────────────────────────────────────────────────────────────────────────────
tic currently comes with support for three CI providers:
● Appveyor
● Circle CI
● Travis CI
There is no need to use all of them. You can choose which one(s) you want to use, whether
you want to deploy (i.e. push from builds) and if you want to test on multiple R versions.

We recommend the following setup:
● Travis CI: Linux
● Travis CI: macOS
● Appveyor: Windows

── Setting up the CI providers. ─────────────────────────────────────────────────────────────
Next we are getting the selected CI providers ready for deployment. This requires some
interaction with their API and you may need to create an API token.

─────────────────────────────────────────────────────────────────────────────────────────────
Error: `use_tic()` needs the `circle` package. Please install it using `remotes::install_github("pat-s/circle")`.
```

I don't really understand why given the defaults, but have no time to explore more right now.

**Comment by maelle** [comment]
Btw the first sentence about the wizard should depend on the wizard parameter. :-)

**Comment by pat-s** [comment]
Ah now I see. Good catch. One should not call `tic::use_tic(wizard=FALSE)` without a selection of the args beforehand. At least that's how it was for the current revision you were testing.

I now changed the defaults. Circle CI is used as the default for Linux and with that we require the Circle package.

Beforehand, we had a vector of options as defaults. I think here it was a bit unclear that the default was and I am not even sure that it would have worked this way.

Feel free to have another stab at it now :)

**Comment by pat-s** [comment]
Assumed to be solved in 83d9bd41a83f50763b6fbb99f163c9aafc05b6a1 :)
